Library/ViewModels/LiveStreamContainerViewModel.swift

```diff
@@ -104,18 +104,22 @@ LiveStreamContainerViewModelInputs, LiveStreamContainerViewModelOutputs {
     let initialEvent = configData.map { _, event, _, _ in event }
 
     let updatedEventFetch = initialEvent
-      .switchMap { event -> SignalProducer<Event<LiveStreamEvent, LiveApiError>, NoError> in
+      .switchMap { initialEvent -> SignalProducer<Event<LiveStreamEvent, LiveApiError>, NoError> in
 
         timer(interval: .seconds(5), on: AppEnvironment.current.scheduler)
           .prefix(value: Date())
           .flatMap { _ in
             AppEnvironment.current.liveStreamService
-              .fetchEvent(eventId: event.id, uid: AppEnvironment.current.currentUser?.id)
+              .fetchEvent(eventId: initialEvent.id, uid: AppEnvironment.current.currentUser?.id)
               .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
               .materialize()
           }
           .filter { event in
-            event.value?.liveNow == .some(true) || event.value?.hasReplay == .some(true) || event.error != nil
+            if initialEvent.liveNow && event.error == nil {
```

Comment:
yeah, it does seem this multiline `filter` could be replaced by just 

```
(initialEvent.liveNow && event.value?.liveNow == .some(true)) || !initialEvent.liveNow
```

and using a lil bit of boolean logic that simplifies to:

```
event.value?.liveNow == .some(true) || !initialEvent.liveNow
```

i believe that's all correct, but hopefully the tests confirm!